requirements.txt

```diff
@@ -1,14 +1,15 @@
 #
 # This file is autogenerated by pip-compile
 # To update, run:
-#   
-#    pip-compile from pip-tools in requirements.dev.txt
+#
 #    pip-compile --output-file=requirements.txt requirements.in
 #
 joblib==0.16.0            # via scikit-learn
 numpy==1.18.4             # via -r requirements.in, pandas, scikit-learn, scipy
 pandas==1.1.1             # via -r requirements.in
 plotly==4.9.0             # via -r requirements.in
+py4j==0.10.9              # via pyspark
```

Comment:
What is this one for? I thought we just needed pyspark

Reply:
I have no idea haha it was added when I updated with the command.